### ISSUE TITLE
Group "sgx" already exists.

### ISSUE THREAD

**Issue by Goose29** [okd-project/okd-scos#9]
When creating a new cluster, the following is seen on the `systemd-sysusers` unit on all nodes.

```
systemd[1]: Starting Create System Users...
systemd-sysusers[779]: Creating group 'sgx' with GID 991.
systemd-sysusers[779]: Creating group 'systemd-oom' with GID 990.
systemd-sysusers[779]: Creating user 'systemd-oom' (systemd Userspace OOM Killer) with UID 990 and GID 990.
systemd-sysusers[779]: /etc/gshadow: Group "sgx" already exists.
systemd[1]: systemd-sysusers.service: Main process exited, code=exited, status=1/FAILURE
systemd[1]: systemd-sysusers.service: Failed with result 'exit-code'.
systemd[1]: Failed to start Create System Users.
```

This doesn't affect the cluster health, as mentioned here: https://github.com/okd-project/okd-scos/issues/4 but it is still present in the latest release.

**Comment by LorbusChris** [comment]
This issue should be fixed by https://github.com/coreos/rpm-ostree/pull/4503 which landed in rpm-ostree v2023.6.
Please notify here if the issue still persists.

**Comment by AlbertoPimpo** [comment]
> This issue should be fixed by [coreos/rpm-ostree#4503](https://github.com/coreos/rpm-ostree/pull/4503) which landed in rpm-ostree v2023.6. Please notify here if the issue still persists.

@LorbusChris just installed 4.16.0-0.okd-scos-2024-09-27-110344 (upgrading from 4.15.0-0.okd-2024-03-10-010116) and I had the same issue.
